packages/dds/tree/src/test/domains/json/jsonCursor.bench.ts

```diff
@@ -202,9 +261,41 @@ function extractAvgValsFromTwitter(
     cursor.exitField();
 }
 
+function extractCoordinatesFromCanadaDirect(
+    // TODO: export Canada type and use instead of any.
```

Comment:
I think the type can just be exported since the twitter one is. @seanimam for confirmation.